aim/config.py

```diff
@@ -84,8 +72,12 @@
     cfg.BoolOpt('recovery_restart', default=True,
                 help=("Set to True if you want the agents to exit in critical "
                       "situations.")),
+    cfg.StrOpt('aim_service_identifier', default=socket.gethostname(),
+               help="(NODB) Identifier for this specific AID service, "
```

Comment:
If the important thing about these options is whether restart is required or not, then the help string should mention that instead of what looks like an implementation detail. Also, as I understand, the help string is for human consumption, so perhaps human readability should be given more weightage.